pkgs/top-level/python-packages.nix

```diff
@@ -8658,7 +8658,19 @@ in {
     };
   };
 
-  pytorch = callPackage ../development/python-modules/pytorch { };
+  pytorch = callPackage ../development/python-modules/pytorch {
+    cudaSupport = pkgs.config.cudaSupport or false;
+    cudatoolkit = pkgs.cudatoolkit;
+    cudnn = pkgs.cudnn;
+  };
+
+  pytorchWithCuda = self.pytorch.override {
```

Comment:
why add extra attributes when its a matter of overriding and setting `cudaSupport=true;`?